h/static/styles/partials-v2/_search.scss

```diff
@@ -117,18 +114,20 @@
 }
 
 .search-result-bucket__annotation-cards {
-  flex-grow: 1;
   max-width: 500px;
+  flex-shrink: 0;
   padding-left: 0;
   padding-right: 0;
 }
 
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
   margin-left: 30px;
   word-wrap: break-word;
```

Comment:
Good idea. `word-wrap: break-all` seems like a good idea for URIs, for example without:

![screenshot from 2016-10-18 11-08-06](https://cloud.githubusercontent.com/assets/22498/19473544/07761440-9524-11e6-9faa-5232edc8c17a.png)

With:

![screenshot from 2016-10-18 11-14-12](https://cloud.githubusercontent.com/assets/22498/19473553/0f2fb16e-9524-11e6-9117-bb1fbf6d30dc.png)

I'd say that we should also use `break-all` for usernames since usernames can't contain spaces anyway (although as you say this is far less likely to ever actually make a difference for usernames).

Tags _can_ contain spaces so I think stick with `word-break: normal` for those (though I haven't extensively checked to see how this works with production usage of tags, which probably various widely between different groups of users).

I'll add a commit for this...